.gitignore

```diff
@@ -127,6 +127,3 @@ dmypy.json
 
 # Pycharm project settings
 .idea/
-
-# Model specification files
-src/vivarium_census_prl_synth_pop/model_specifications/*.yaml
```

Comment:
We don't want to change this. Custom model specs shouldn't get added. You shouldn't need this line to add the sample model spec.

Reply:
How do I add it w/o not ignoring it?

Reply:
Ah git add --force. Well I cheated it by just undoing the gitignore.